frontend/lang/messages/en-US.json

```
@@ -596,12 +596,12 @@
     "create-recipe-description": "Create a new recipe from scratch.",
     "create-recipes": "Create Recipes",
     "import-with-zip": "Import with .zip",
-    "create-recipe-from-an-image": "Create Recipe from an Image",
-    "create-recipe-from-an-image-description": "Create a recipe by uploading an image of it. Mealie will attempt to extract the text from the image using AI and create a recipe from it.",
+    "create-recipe-from-an-image": "Create Recipe from Image(s)",
+    "create-recipe-from-an-image-description": "Create a recipe by uploading images of it. Mealie will attempt to extract the text from the image(s) using AI and create a recipe from it.",
     "crop-and-rotate-the-image": "Crop and rotate the image so that only the text is visible, and it's in the correct orientation.",
-    "create-from-image": "Create from Image",
+    "create-from-image": "Create from Image(s)",
```

Might want to just remove `create-from-image` and replace it with `create-from-images`. Will be less confusing in the application code.

```suggestion
    "create-from-images": "Create from Images",
```